scss/base/4-elements/_typography.scss

```diff
@@ -1,51 +1,82 @@
 h1,
 .h1 {
-    font-size: 3em;
-    line-height: 1.25em;
-    font-weight: $cads-font-weight__extra-bold;
-    color: $cads-palette__heritage-blue;
+    font-size: 2.5em;
+    line-height: 1.2em;
+    font-weight: $cads-font-weight__bold;
+    color: $cads-palette__black;
     margin: 0;
     padding: 1em 0 1em 0;
```

Comment:
Are we leaving the spacing above/below?

Reply:
Spacing still needs to be worked out / designed so we will add that as a separate pull request.